### PR TITLE
Removed fission reference

### PR DESCRIPTION
**Pull request by therahulbhati**
Signed-off-by: therahulbhati <rjbhati009@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ The job of the connector is to read messages from the topic, invoke a HTTP endpo
 
 ```
 {
-   "X-Fission-MQTrigger-Topic": Topic,
-   "X-Fission-MQTrigger-RespTopic": ResponseTopic,
-   "X-Fission-MQTrigger-ErrorTopic": ErrorTopic,
+   "Topic": Topic,
+   "RespTopic": ResponseTopic,
+   "ErrorTopic": ErrorTopic,
```
Comment (vishal-biyani) [inline README.md:38]: Although the X header names are deprecated, these headers can easily collide with something else. I suggest some format like `x-keda-topic` or just `keda-source-topic`

Comment (vishal-biyani) [reply README.md:38]: Of course, we will have to change in the actual connector code I think?

Comment (therahulbhati) [reply README.md]: Yes makes sense

File: kafka-http-connector/main.go [hunk 2]
```diff
@@ -213,11 +213,11 @@ func (conn *kafkaConnector) ConsumeClaim(session sarama.ConsumerGroupSession, cl
...
 		headers := http.Header{
-			"Topic":        {conn.connectorData.Topic},
-			"RespTopic":    {conn.connectorData.ResponseTopic},
-			"ErrorTopic":   {conn.connectorData.ErrorTopic},
-			"Content-Type": {conn.connectorData.ContentType},
-			"Source-Name":  {conn.connectorData.SourceName},
+			"KEDA-Topic":          {conn.connectorData.Topic},
+			"KEDA-Response-Topic": {conn.connectorData.ResponseTopic},
+			"KEDA-Error-Topic":    {conn.connectorData.ErrorTopic},
+			"KEDA-Content-Type":   {conn.connectorData.ContentType},
```
Comment (vishal-biyani) [inline kafka-http-connector/main.go:219]: `Content-Type` is a generic HTTP header, let's not change that :)  (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type)

Comment (therahulbhati) [reply kafka-http-connector/main.go]: Done

**Pull request merged**